apps/remix-ide-e2e/src/tests/proxy.test.ts

```diff
@@ -0,0 +1,289 @@
+    browser
+      .addFile('myTokenV1.sol', sources[0]['myTokenV1.sol'])
+      .clickLaunchIcon('solidity')
+      .pause(2000)
+      .click('[data-id="compilerContainerCompileBtn"]')
+      .waitForElementPresent('select[id="compiledContracts"] option[value=MyToken]', 60000)
+      .clickLaunchIcon('udapp')
+      .click('select.udapp_contractNames')
+      .click('select.udapp_contractNames option[value=MyToken]')
+      .waitForElementPresent('[data-id="contractGUIDeployWithProxyLabel"]')
+      .waitForElementPresent('[data-id="contractGUIUpgradeImplementationLabel"]')
+  },
+
+  'Should show upgrade proxy option for child contract inheriting UUPS parent contract': function (browser: NightwatchBrowser) {
+    browser
+      .addFile('myTokenV2.sol', sources[1]['myTokenV2.sol'])
+      .clickLaunchIcon('solidity')
+      .pause(2000)
+      .click('[data-id="compilerContainerCompileBtn"]')
+      .waitForElementPresent('select[id="compiledContracts"] option[value=MyTokenV2]', 60000)
+      .clickLaunchIcon('udapp')
+      .click('select.udapp_contractNames')
+      .click('select.udapp_contractNames option[value=MyTokenV2]')
+      .waitForElementPresent('[data-id="contractGUIDeployWithProxyLabel"]')
+      .waitForElementPresent('[data-id="contractGUIUpgradeImplementationLabel"]')
+  },
+
+  'Should deploy proxy without initialize parameters': function (browser: NightwatchBrowser) {
+    browser
+      .openFile('myTokenV1.sol')
+      .clickLaunchIcon('solidity')
+      .pause(2000)
+      .click('[data-id="compilerContainerCompileBtn"]')
+      .waitForElementPresent('select[id="compiledContracts"] option[value=MyToken]', 60000)
+      .clickLaunchIcon('udapp')
+      .click('select.udapp_contractNames')
+      .click('select.udapp_contractNames option[value=MyToken]')
+      .waitForElementPresent('[data-id="contractGUIDeployWithProxyLabel"]')
+      .click('[data-id="contractGUIDeployWithProxyLabel"]')
+      .createContract('')
+      .waitForElementContainsText('[data-id="udappNotifyModalDialogModalTitle-react"]', 'Deploy Implementation & Proxy (ERC1967)')
+      .waitForElementVisible('[data-id="udappNotify-modal-footer-ok-react"]')
+      .click('[data-id="udappNotify-modal-footer-ok-react"]')
+      .waitForElementContainsText('[data-id="confirmProxyDeploymentModalDialogModalTitle-react"]', 'Confirm Deploy Proxy (ERC1967)')
+      .waitForElementVisible('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .click('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander0"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander1"]')
+  },
+
+  'Should interact with deployed contract via ERC1967 (proxy)': function (browser: NightwatchBrowser) {
+    browser
+      .getAddressAtPosition(1, (address) => {
+        console.log('Proxy address: ' + address)
```

Comment:
log should be removed

Reply:
Console logs should be left for easy debugging in circleci.